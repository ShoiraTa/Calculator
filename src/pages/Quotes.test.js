import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quotes';

test('matches home snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
