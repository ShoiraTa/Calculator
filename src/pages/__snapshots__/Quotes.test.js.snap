// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches home snapshot 1`] = `
<div>
  <div
    className="container"
  >
    <h1>
      \`Pure mathematics is, in its way, the poetry of logical ideas. â€”
      <br />
      <span>
        Albert Einstein, German theoretical physicist
      </span>
      \`
    </h1>
  </div>
</div>
`;
